mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSConcept.kt

```diff
@@ -72,8 +80,12 @@ data class MPSConcept(val concept: SAbstractConceptAdapter) : IConcept {
     }
 
     override fun isExactly(concept: IConcept?): Boolean {
-        val otherMpsConcept = concept as? MPSConcept ?: return false
-        return this.concept == otherMpsConcept.concept
+        if (concept == null) return false
+        if (concept is MPSConcept) {
```

Comment:
**Question:** Before checking for equality, why do we have to check first if it's an `MPSConcept`?

Reply:
Removed the unnecessary instance of check.